doc/input-output-reference/src/overview/group-airflow.tex

```diff
@@ -1884,6 +1931,22 @@ \subsubsection{Outputs}\label{zoneearthtube-outputs}
 
 This is the humidity ratio of the air entering the zone after passing through the earth tube {[}kgWater/kgDryAir{]}.
 
+\paragraph{Earth Tube Node Temperature <X> {[}C{]}}\label{earth-tube-node-temperatures-c}
+
+This will generate the internal node temperatures {[}C{]} for the earth tube as a result of the 1-D finite difference model.  This is only valid for the 1-D (Vertical) model.
+
+\paragraph{Earth Tube Undisturbed Ground Temperature <X> {[}C{]}}\label{earth-tube-undisturbed-node-temperatures-c}
+
+This will report the theoretical undisturbed ground temperature at the location of the internal node temperatures {[}C{]} as well as the average for the location of the earth tube for the 1-D finite difference model.  This is only valid for the 1-D (Vertical) model and provides a comparison of conditions for the simple model and the 1-D (Vertical) model.
+
+\paragraph{Earth Tube Upper Boundary Ground Temperature {[}C{]}}\label{earth-tube-upper-boundary-temperature-c}
+
+This will report the theoretical undisturbed ground temperature at the upper boundary for the 1-D finite difference model.  This is only valid for the 1-D (Vertical) model.
+
+\paragraph{Earth Tube Lower Boundary Ground Temperature {[}C{]}}\label{earth-tube-upper-boundary-temperature-c}
```

Comment:
GitHub's doc build is complaining about the multiply defined label here.  Need the relabel as -lower-boundary ...

Reply:
Oh, rats.  Copied and pasted and then forgot to update that part of the statement.  I'll fix that (hopefully first thing tomorrow).